### PR TITLE
Updated Readme Files

### PR DESCRIPTION
**Pull request by yakovmanshin**
I updated [readme.html](https://github.com/YOURLS/YOURLS/blob/master/readme.html) to switch links to HTTPS versions, add a link to installation guide for a newer version of Ubuntu, and correct grammar mistakes.

In [README.md](https://github.com/YOURLS/YOURLS/blob/master/README.md), I switched links to HTTPS versions and added minor improvements.

### CODE REVIEW

**Comment by ozh** [comment]
Changing all http to https where applicable is indeed a neat fix. I'm OK with all "minor improvements", except one: you suggest changing "Sequential keyword" with "Random keyword. The thing is, "sequential" is the correct term here. Out of the box, generated keywords are sequential. With a plugin, you can make them random (or any other behavior)

**Comment by yakovmanshin** [comment]
I guess I’ll revert to “sequential” then.

File: readme.html [hunk 1]
```diff
@@ -344,9 +344,10 @@ <h2 class='first' id="fresh">Fresh Install</h2>
...
 			<ul>
-            <li><a href="https://www.rosehosting.com/blog/how-to-install-yourls-on-ubuntu-14-04/">How to Install YOURLS on Ubuntu 14.04</a></li>
-			<li><a href="https://thishosting.rocks/how-to-shorten-your-links-with-your-own-domain/">How to Install YOURLS on a VPS or shared hosting</a></li>
-			<li><a href="https://www.websitehostingrating.com/how-to-install-yourls-using-softaculous/">How to Install YOURLS using Softaculous</a></li>
+                <li><a href="https://www.rosehosting.com/blog/how-to-install-yourls-on-ubuntu-14-04/">How to Install YOURLS on Ubuntu 14.04</a></li>
+                <li><a href="https://computingforgeeks.com/how-to-install-yourls-your-own-url-shortener-on-ubuntu-18-04-bionic-beaver-linux/">How to install YOURLS on Ubuntu 18.04</a></li>
+                <li><a href="https://thishosting.rocks/how-to-shorten-your-links-with-your-own-domain/">How to Install YOURLS on a VPS or shared hosting</a></li>
+                <li><a href="https://www.websitehostingrating.com/how-to-install-yourls-using-softaculous/">How to Install YOURLS using Softaculous</a></li>
```
Comment (dgw) [inline readme.html]: This block is indented with spaces, but the rest of the file uses tabs.

**Comment by dgw** [comment]
@ozh I'd say this should be squashed down to the initial commit once the little details are fixed. What say you?

**Comment by dgw** [comment]
@yakovmanshin Would you mind squashing (as fixup) those last two commits? Then I'll merge this when I get home. :smile_cat:

**Comment by yakovmanshin** [comment]
@dgw Sure, squash them.

**Comment by dgw** [comment]
@yakovmanshin Did the rebase/squash, but can't push it to your branch—I get "permission denied". Perhaps "Allow edits from maintainers" needs to be enabled ([see GitHub docs](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)), and/or there's some branch restriction at play (such as GitHub not allowing me to push to your fork's default branch).

Hopefully this is as simple as checking the box to allow edits, but in case it isn't I did push the squashed history to [dgw@2472-rebased](https://github.com/dgw/YOURLS/tree/2472-rebased) so it's available for you to pull and force-push yourself.

**Comment by yakovmanshin** [comment]
@dgw Wow, I don’t even know how this checkbox got disabled. I enabled it, so you can try again.

**Comment by dgw** [comment]
@yakovmanshin Maybe it's disabled by default if you open a PR from a fork's default branch? Just speculating; haven't been able to find a whole lot of documentation except what I linked earlier.

Thanks for checking the box; GitHub accepted the push now, and I'll merge when Travis is happy. :+1:

**Comment by yakovmanshin** [comment]
@dgw That’s great; thank you!

**Pull request merged**